documentation/asciidoc/computers/camera/raspicam.adoc

```diff
@@ -1,3 +1,46 @@
+== Raspicam applications
+
+[WARNING]
+====
+Raspberry Pi has transitioned from a legacy camera software stack based on proprietary Broadcom GPU code to an open-source stack based on `libcamera`. As such, the _Raspicam_ stack is now deprecated.  Raspberry Pi OS images from _Bullseye_ onwards will contain *only* the `libcamera`-based stack. Raspberry Pi OS images up to and including _Buster_ will contain the legacy _Raspicam_ stack, though the `libcamera` stack and applications can be installed xref:camera_software.adoc#libcamera-and-libcamera-apps-packages[using _apt_], or built by following the xref:camera_software.adoc#building-libcamera-and-libcamera-apps[normal build instructions].
+====
+
+Users are encouraged to use the newest OS images and the `libcamera`-based stack because:
+
+* It will continue to be developed moving forward.
+* Raspberry Pi and 3rd parties can fix bugs and problems in the camera stack.
+* Raspberry Pi and 3rd parties can add new features to the camera stack.
+* It is much easier to add support for new cameras.
+* 3rd parties can add support directly for their own cameras.
+* Nearly all aspects of the camera tuning can be changed by users.
+* It integrates much more conveniently with other standard Linux APIs.
+* Raspberry Pi supply a set of `libcamera-apps` which emulate most of the features of the legacy applications.
+* It provdes a feature-rich post-processing framework integrating OpenCV and TensorFlow Lite.
+* Libcamera makes it easier to control the parameters of the image sensor and the camera system.
+* It is fully supported on 64-bit operating systems.
+
+Reasons to consider staying with an older OS and using the legacy _Raspicam_ stack might include:
+
+* It may perform better on Raspberry Pi 2 and Raspberry Pi Zero devices, as it offloads more to the GPU and is less dependent on the ARM cores.
+* `libcamera` is still missing certain features, most notably Python bindings. Whilst this work is in progress, users who need a Python interface (such as _Picamera_) will have to stay with the legacy stack for the time being.
```

Comment:
I guess this bit about the missing Python bindings is outdated now?

Reply:
True.  I'll remove that sentence completely.  There is a up-coming task about adding some details on picamera2 to these pages.